app/templates/compare_revisions.html

```diff
@@ -26,13 +26,10 @@
   <span class="govuk-caption-l">{{ service['supplierName'] }}</span>
   <h1 class="govuk-heading-l">{{ service['serviceName'] }}</h1>
 
-  {%
-    with
-      url = url_for("external.direct_award_service_page", framework_family=service['frameworkFamily'], service_id=service["id"]),
-      text = "View service"
-  %}
-    {% include "toolkit/secondary-action-link.html" %}
-  {% endwith %}
+  <a class="govuk-link govuk-!-margin-bottom-3 govuk-!-display-inline-block" href="{{ url_for("external.direct_award_service_page", framework_family=service['frameworkFamily'], service_id=service["id"]) }}">
```

Comment:
Can I be really picky and ask that we use single quotes around the `url_for` param if we're using double quotes on the `href`? For the sake of the colour scheme on my editor, if nothing else 😹

Reply:
Oh nuts yes that's a good point